resources/options_lookup.tsv

```diff
@@ -12714,4 +12754,4 @@ AIANNH Area	No
 AIANNH Area	Yes																									
 Household Has Tribal Persons	No																									
 Household Has Tribal Persons	Yes																									
-Household Has Tribal Persons	Not Available																									
+Household Has Tribal Persons	Not Available																									
```

Comment:
Lost the newline here?

Reply:
Yes, but added a new characteristic in PR #1080 that changes the last couple lines of options lookup.